recipe/meta.yaml

```
@@ -1,47 +1,62 @@
 {% set name = "PyGithub" %}
-{% set version = "1.55" %}
+{% set version = "2.4.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 1bbfff9372047ff3f21d5cd8e07720f3dbfdaf6462fcaed9d815f528f1ba7283
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name|lower }}-{{ version }}.tar.gz
+  sha256: 6601e22627e87bac192f1e2e39c6e6f69a43152cfb8f307cee575879320b3051
 
 build:
-  noarch: python
-  number: 1
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  number: 0
+  skip: True  # [py<38]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
-    - python >=3.6
+    - python
     - pip
-
+    - wheel
+    - setuptools
+    - setuptools-scm
   run:
-    - python >=3.6
-    - pyjwt >=2.0
+    - python
+    - pyjwt >=2.4.0
```

```suggestion
    # pyjwt[crypto] is pyjwt + cryptography
    - pyjwt >=2.4.0
    - cryptography
```
See 
https://github.com/PyGithub/PyGithub/blob/v2.4.0/pyproject.toml#L33C4-L33C9
https://github.com/jpadilla/pyjwt/blob/master/setup.cfg#L49C5-L49C17
https://github.com/AnacondaRecipes/pyjwt-feedstock/blob/master/recipe/meta.yaml#L26-L27